doc/setup/quickstart_cmake/index.md

```diff
@@ -139,38 +142,36 @@ make
 ./hello_world
 ```
 
-The output should be `Hello world`. Note that the build type is specified with `-DCMAKE_BUILD_TYPE=Release`. 
-Specifying `Release` enables an optimized build where no debug information is available. Release mode is therefore 
+The output should be `Hello World!`. Note that the build type is specified with `-DCMAKE_BUILD_TYPE=Release`.
+Specifying `Release` enables an optimized build where no debug information is available. Release mode is therefore
 suitable for the end user. There is an example of a `Debug` build that is suitable for contributors in the next tutorial.
```

Comment:
Can we name and link to the tutorial? We might change the order at some point and this would be outdated when :)

Reply:
I rephrased it a bit and now link to the tutorial landing page, because there is not a tutorial that says "build with Debug".
We should have a look at doing landing pages for the documentation (tutorial, about, etc.), because you can only link via the usergroup.html and this is not stable